app/src/main/java/com/karhoo/karhootraveller/presentation/splash/register/SplashPresenter.kt

```diff
@@ -102,6 +103,11 @@ internal class SplashPresenter(view: SplashMVP.View,
     }
 
     override fun handleLoginTypeSelection(loginType: String) {
+        Log.d("Adyen", userStore.paymentProvider.toString())
```

Comment:
Shall we use a more generic e.g. Payment instead of Adyen

Reply:
Oops! Meant to delete that!